bukkit/src/main/java/com/denizenscript/depenizen/bukkit/extensions/residence/ResidenceLocationExtension.java

```diff
@@ -0,0 +1,83 @@
+package com.denizenscript.depenizen.bukkit.extensions.residence;
+
+import com.bekvon.bukkit.residence.Residence;
+import com.bekvon.bukkit.residence.protection.ClaimedResidence;
+import com.denizenscript.depenizen.bukkit.extensions.dObjectExtension;
+import com.denizenscript.depenizen.bukkit.objects.residence.dResidence;
+import net.aufdemrand.denizen.objects.dLocation;
+import net.aufdemrand.denizencore.objects.Element;
+import net.aufdemrand.denizencore.objects.Mechanism;
+import net.aufdemrand.denizencore.objects.dObject;
+import net.aufdemrand.denizencore.tags.Attribute;
+
+public class ResidenceLocationExtension extends dObjectExtension {
+
+    public static boolean describes(dObject object) {
+        return object instanceof dLocation;
+    }
+
+    public static ResidenceLocationExtension getFrom(dObject object) {
+        if (!describes(object)) {
+            return null;
+        }
+        else {
+            return new ResidenceLocationExtension((dLocation) object);
+        }
+    }
+
+    ///////////////////
+    // Instance Fields and Methods
+    /////////////
+
+    public static final String[] handledTags = new String[] {
+            "has_residence", "residence"
+    };
+
+    public static final String[] handledMechs = new String[] {
+    }; // None
+
+    private ResidenceLocationExtension(dLocation location) {
+        this.location = location;
+    }
+
+    dLocation location;
+
+    @Override
+    public String getAttribute(Attribute attribute) {
+        // <--[tag]
+        // @attribute <l@location.has_residence>
+        // @returns Element(Boolean)
+        // @description
+        // Returns if the location has a residence.
+        // @Plugin DepenizenBukkit, Residence
+        // -->
+        if (attribute.startsWith("has_residence")) {
+            org.bukkit.Location loc = new org.bukkit.Location(location.getWorld(), location.getBlockX(), location.getBlockY(), location.getBlockZ(), location.getYaw(), location.getPitch());
```

Comment:
What even is this ^

Reply:
That's the same way I've used to convert Denizen's location object to Bukkit's Object. I used the method that was implemented for PlotMe Location Extension.

Reply:
Well it's wrong in all of those. `dLocation` objects are valid `Location` objects. No conversion at all is needed.